src/Builder/DatagridBuilder.php

```diff
@@ -140,7 +140,11 @@ public function getBaseDatagrid(AdminInterface $admin, array $values = [])
             $defaultOptions['csrf_protection'] = false;
         }
 
-        $formBuilder = $this->formFactory->createNamedBuilder('filter', 'form', [], $defaultOptions);
+        // NEXT_MAJOR: Remove this line when drop Symfony <2.8 support
```

Comment:
I was gonna say support has been dropped, but there is not even a requirement in composer.json. Can you please modify it to add `symfony/form`, which this obviously requires?